src/material-experimental/mdc-checkbox/checkbox.ts

```diff
@@ -348,4 +350,19 @@ export class MatCheckbox implements AfterViewInit, OnDestroy, ControlValueAccess
     this._classes[cssClass] = active;
     this._changeDetectorRef.markForCheck();
   }
+
+  /**
+   * Syncs the indeterminate value with the checkbox DOM node.
+   *
+   * We sync `indeterminate` directly on the DOM node, because in Ivy the check for whether a
+   * property is supported on an element boils down to `if (propName in element)`. Domino's
```

Comment:
Could we file an issue (or send a PR) for Domino and add a TODO here referencing that?

(not sure if there's more to it than adding to the [input attrs object](https://github.com/fgnass/domino/blob/2623f3d72d57865cc697cad8762f2ddff372dd0b/lib/htmlelts.js#L573))